src/generic_core/remote-instance.ts

```diff
@@ -195,19 +189,20 @@ export var remoteProxyInstance :RemoteInstance = null;
           // TODO: Move the logic for resetting the onceSharerCreated promise inside
           // remote-connection.ts.
           this.connection_.resetSharerCreated();
-          this.onceSharerReadyForOffer_ = this.connection_.onceSharerCreated;
+          this.hasSeenOffer = true;
```

Comment:
Here we set this.hasSeenOffer to true, but in the next line we call this.startShare_, which in some cases calls this.stopShare, which in some cases sets this.hasSeenOffer back to false.  Is this going to cause problems the next time we get a CANDIDATE?  i.e. it will think we haven't seen an offer when really we have?